### PR TITLE
Create ega_dataset

### PR DESCRIPTION
**Pull request by ivyEBI**
How the EGA data mapped to bioschema

### CODE REVIEW

**Review by jdylan — COMMENTED** [review]
This is an EGA study not dataset

**Review by ivyEBI — COMMENTED** [review]
update a new example

File: Dataset/examples/ega_study [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<script type="application/ld+json">
+{
+    "name": "Exome-sequencing of human B cell lymphoma cell lines",
+    "description": "26 cell lines derived from human Diffuse Large B Cell lymphomas (DLBCL) or Burkit Lymphomas (BL) were subjected to whole exome sequencing. Exome capture was carried out using the SeqCap EZ Exome Library 2.0 kit (Roche/Nimblegen) and 100 bp single-read sequencing was performed on a HiSeq2500 (Illumina). 82% of the coding region was covered at least 30x.",
+    "url": "https://ega-archive.org/studies/EGAS00001001463",
+    "identifier": [
+        "EGAS00001001463"
+    ],
+    "keywords": "null",
```
Comment (AlasdairGray) [inline Dataset/examples/ega_study:9]: Nulls should not be used

File: Dataset/examples/ega_study [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "distribution": "https://ega-archive.org/studies/EGAS00001001463",
+    "includedInDataCatalog": [
+        "https://ega-archive.org/"
+    ],
+    "license": "null",
+    "measurementTechnique": [
+        "WXS"
+    ],
+    "variableMeasured": [
+        "others"
```
Comment (AlasdairGray) [inline Dataset/examples/ega_study:25]: This is not very informative

File: Dataset/examples/ega_study [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        "http://europepmc.org/abstract/MED/27324824"
+    ],
+    "creator": [
+        "MED-DUS"
+    ],
+    "distribution": "https://ega-archive.org/studies/EGAS00001001463",
+    "includedInDataCatalog": [
+        "https://ega-archive.org/"
+    ],
+    "license": "null",
```
Comment (AlasdairGray) [inline Dataset/examples/ega_study:20]: Null should not be used

**Review by AlasdairGray — CHANGES_REQUESTED** [review]
As @jdylan has said, this markup seems to be describing a Study rather than a Dataset. On the corresponding webpage I see that the Study makes a single Dataset available. Should this page be marked up using the [Study profile](http://bioschemas.org/devSpecs/Study/) (in development) and the [page about the Dataset](https://ega-archive.org/datasets/EGAD00001002262) marked up with the Dataset profile?

**Comment by AlasdairGray** [comment]
@ivyEBI @jdylan  let me know if you need help making the requested changes.

**Comment by AlasdairGray** [comment]
As there has been no action to correct this pull request, we are going to close it.

**Pull request closed without merging**